src/actions/auth.js

```diff
@@ -443,21 +443,35 @@ export const login = (dispatch, firebase, credentials) => {
 
   const { method, params } = getLoginMethodAndParams(firebase, credentials)
 
-  return firebase
-    .auth()
-    [method](...params)
+  const firebaseAuth = firebase.auth()
+
+  let firebaseMethod = firebaseAuth[method]
+
+  // support older versions of firebase that still use signInWithCustomToken & signInWithEmailAndPassword
+  switch (method) {
```

Comment:
Is it possible for us to fit this logic in the `getLoginMethodAndParams` function?